src/freeze/enable.rs

```diff
@@ -0,0 +1,127 @@
+use super::*;
+
+pub struct EnableFreezeArgs {
+    pub keypair: Option<String>,
+    pub rpc_url: Option<String>,
+    pub cache: String,
+    pub config: String,
+    pub candy_machine: Option<String>,
+    pub freeze_days: Option<u8>,
```

Comment:
Perhaps we need to have the `freeze_days` under a flag or as the first argument. I tried to specify the `--cache` and the `freeze_days` but the command tried to parse the argument as the candy machine id:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/729235/190209572-b412d64b-8ed7-48b8-93fc-1e0b719fa647.png">

Reply:
[Fixed.](https://github.com/metaplex-foundation/sugar/pull/345/commits/6bdb79ff9f6e5b53a1014497116eb5ec45a2b478)


![cilantro@pop-os: ~-Sync-sugar_test2_002](https://user-images.githubusercontent.com/1684605/190217978-9e159340-7f0c-451a-ac4b-99e5c72591d8.png)